utils/include/tlv.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+
+/**
+ * @brief Write data to storage.
+ * 
+ * @param ctx Storage context.
+ * @param offset Location where to place the date.
+ * @param data Data to save.
+ * @param data_size Number of bytes to save.
+ * 
+ */
+typedef int (*tlv_storage_write)(void *ctx, uint32_t offset, uint8_t *data, uint32_t data_size);
```

Comment:
its a type, so name should be `tlv_storage_write_t`

Reply:
ok, added suffix for the type